broker-cli/commands/wallet.js

```diff
@@ -352,20 +352,44 @@ async function release (args, opts, logger) {
     if (!ACCEPTED_ANSWERS.includes(answer.toLowerCase())) return
 
     try {
-      await client.walletService.releaseChannels({ market, force })
-    } catch (e) {
-      logger.error(`Failed to release payment channels for ${market}`.red)
+      // We want to keep track if any returned channel had an error while attempting
+      // to be released so that we can display the proper error warnings below
+      let channelHasError = false
+
+      const { channels = [] } = await client.walletService.releaseChannels({ market, force })
+
+      if (!channels) {
+        throw new Error('No information was retrieved from the daemon')
+      }
+
+      logger.info(`Released Market: ${market}`)
 
-      if (!force) {
-        logger.info('You can force the release of funds using the `--force` option, however')
-        logger.info('this has the potential to lock your funds for an extended period of time')
-        logger.info('and cost additional fees')
+      channels.forEach((channel) => {
+        const { symbol, error, status } = channel
+
+        if (error) {
+          channelHasError = true
+          logger.info(`${symbol}: ` + `${status}: ${error}`.red)
+        } else {
+          logger.info(`${symbol}: ` + status.green)
+        }
+      })
+
+      if (channelHasError) {
+        logger.info(`\nErrors have occurred while trying to release channels for ${market}:`.red)
       }
 
+      if (channelHasError && !force) {
+        logger.info('If the error above suggests an uncooperative closing of any channels on')
```

Comment:
My comment from before still applies I think: is there any way to know if the error suggests an uncooperative close?

I'm imagining an output like this:

```
BTC: Released
LTC: Error: Peer is unavailable. You must wait for the peer to come online or perform an uncooperative close.

Errors have occurred while trying to release channels for BTC/LTC:
If the error above suggests an uncooperative closing of any channels on
your daemon, you can force the release of funds using the --force option.
```

Doesn't that seem a little weird?

Reply:
@treygriffith the user has to make the decision to force close the channels, so RE: showing a message, there is no way around it. However, matching the error is an uphill battle, the only way we could potentially identify the error is with string matching

Reply:
@treygriffith I went for it and made the change. it looks good